src/App/Fossa/Analyze.hs

```diff
@@ -390,6 +394,7 @@ analyze cfg = Diag.context "fossa-analyze" $ do
           (Just firstParty, Nothing) -> Just firstParty
   let keywordSearchResultsFound = (maybe False (not . null . lernieResultsKeywordSearches) lernieResults)
   let outputResult = buildResult includeAll additionalSourceUnits filteredProjects' licenseSourceUnits
+  logDebug $ "Source unit result: " <> pretty (show outputResult)
```

Comment:
The source units are already output in in the debug bundle under `bundleOutput` [here](https://github.com/fossas/fossa-cli/blob/master/src/App/Fossa/Analyze/Debug.hs#L124) I think. Is there a reason to output them here in addition to that? 

More generally, I'd consider putting some of these data as top-level keys in the [`DebugBundle`](https://github.com/fossas/fossa-cli/blob/master/src/App/Fossa/Analyze/Debug.hs#L165) or sub-keys of `bundleOutput`. They'd be easier to locate I think if that were the case.

Reply:
Good suggestion. I've removed this extraneous message.

Is it alright to merge the others as messages for now? I'd like to get this into user hands ASAP, and looking at the plumbing around `collectDebugBundle` it looks like it might take some more thoughtful plumbing to get analysis messages out of the `Result`.

Reply:
Merging given Slack conversation.